src/mgr/DaemonServer.cc

```diff
@@ -2545,14 +2545,26 @@ bool DaemonServer::_handle_command(
     return true;
   }
 
+  // Validate that the module is enabled
+  auto& py_handler_name = py_command.module_name;
+  PyModuleRef module = py_modules.get_module(py_handler_name);
+  ceph_assert(module);
```

Comment:
We haven't done any validation on the module name yet, have we? So if there's a non-existent module I think this just crashes the mgr?

Reply:
Right, I've pushed an update that should rectify that I hope. PTAL.